tests/bats/_helpers.bash

```diff
@@ -0,0 +1,279 @@
+#!/usr/bin/env bash
```

Comment:
common helpers. these were tested in DrevOps project (have their own tests). 
https://github.com/drevops/drevops/blob/8.x/tests/bats/helpers.bats